### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by benhyland**
Covers the basic structure, entry point, config options.
The structure the old readme referred to has been deleted.

### CODE REVIEW

**Comment by tijmenb** [comment]
Could you use the README structure from the styleguides? 

https://github.com/alphagov/styleguides/blob/master/use-of-READMEs.md

File: README.md [hunk 1]
```diff
@@ -1,39 +1,31 @@
...
-with [rummager](https://github.com/alphagov/rummager) data.
+- Query Rummager and Publishing API public apis and import their data into an in-memory sqlite db
+- Run several independent checks against the local db, each of which writes a csv output file in the working directory.
 
-### Usage
+The exit code for a run is `1` if any check fails, `0` otherwise.
 
-You can simply run the following
+Each check can filter its usual output using a whitelist. This allows us to run the checks automatically
+in Jenkins and still keep the job green even though there may me some known problems we are working on.
```
Comment (malcolmbaig) [inline README.md:19]: "may be"

**Comment by malcolmbaig** [comment]
One typo, ditto @tijmenb's comment re: structure from the stylguides. 

Otherwise, content looks fine.

**Comment by benhyland** [comment]
@tijmenb @mobaig thanks, I think your comments have been addressed

**Comment by tijmenb** [comment]
👍 excellent!

**Comment by malcolmbaig** [comment]
~~@benhyland my typo wasn't ; ).~~

NVM, just seen the new commit. 👍

**Comment by benhyland** [comment]
@mobaig yep, confusion was due to my git command line fingerfailure. Thanks for chasing :)

**Pull request merged**